R/ospsuite-env.R

```diff
@@ -60,5 +60,11 @@ getOSPSuiteSetting <- function(settingName){
     stop(messages$errorOSPSuiteSettingNotFound(settingName))
   }
 
-  return(ospsuiteEnv[[settingName]])
+  obj <- ospsuiteEnv[[settingName]]

```

Comment:
A workaround to always return a value, and not a function.

Reply:
ah ah this is kind of badass cool